docs/Use With Phoenix.md

```diff
@@ -0,0 +1,333 @@
+# Using Distillery With Phoenix
+
+**IMPORTANT**: Before starting, please review the [Terminology](https://hexdocs.pm/distillery/terminology.html)
+page to get familiar with the terms used in this guide.
+
+This is a simple guide on how to use Distillery with a production
+config with [Phoenix](http://www.phoenixframework.org/). Here we will cover:
+
+- Adding Distillery to your Phoenix application
+- Initializing Distillery
+- Configuring your release
+- Building your release
+- Deploying your release
+- Building an upgrade release
+- Deploying an upgrade
+
+It is expected that you will have:
+
+- a working Phoenix application
+- a working `config/prod.exs`
+- a _build environment which matches your production environment_
+- your static asset path is the default `priv/static`
+
+**NOTE**: It does not make much sense to cut a release for a `dev`
+configuration with Phoenix, so this guide assumes you are interested
+in `prod` releases.
+
+Let's get started!
+
+## Adding Distillery to your project
+
+Just add the following to your deps list in `mix.exs`:
+
+```elixir
+defp deps do
+  [{:distillery, "~> 0.9"}]
```

Comment:
May want to make this `~> MAJ.MIN` so we don't have to update this all the time.